### ISSUE TITLE
TypeError: predict() missing 1 required positional argument: 'model_or_model_path'

### ISSUE THREAD

**Issue by drscotthawley** [spotify/basic-pitch#121]
**Describe the bug**
When I executed the suggested Python code on the README.md file...
```python
from basic_pitch.inference import predict
from basic_pitch import ICASSP_2022_MODEL_PATH

model_output, midi_data, note_events = predict('my_audio.wav')
```
I receive this Error message: 

```python
    model_output, midi_data, note_events = predict(tmp_audio_file)
TypeError: predict() missing 1 required positional argument: 'model_or_model_path'
```

Looking at [line 416 of inference.py](https://github.com/spotify/basic-pitch/blob/5fc20ddfd1328969d235b9c155ba3304622b2f50/basic_pitch/inference.py#L416), it appears the Python REPL is not wrong: `model_or_model_path` *is* a required argument for `predict()`:

```python
def predict(
    audio_path: Union[pathlib.Path, str],
    model_or_model_path: Union[Model, pathlib.Path, str],
...
```

How then is the documentation on the README to be applied? 

**To Reproduce**
1. Try to use the code from the README
2. Supply an audio file
3. Execute the code

**Expected behavior**
No error, execution would be smooth

**Screenshots**
<img width="1072" alt="Screenshot 2024-04-15 at 9 00 17 PM" src="https://github.com/spotify/basic-pitch/assets/13925685/5c6a9d0e-447c-4e52-8c71-a62771efc49f">

**Desktop (please complete the following information):**
 - OS: MacOs
 - Browser n/a

**Smartphone (please complete the following information):**

**Additional context**
```
$ pip list | grep basic
basic-pitch               0.3.0

$ python --version
Python 3.10.12

```

**Comment by drscotthawley** [comment]
Unrelated but it affects my ability to verify the Python functionality: 
 the demo website appears to be broken.  

Try uploading or recording any audio, in Brave, Chrome or Safari.  After the audio loads, the upload pane resets without any MIDI appearing. JavaScript Console shows many uncaught "promises", for things like "gain":

```js
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'gain')
    at index-5eb44d2184e903c2.js:1:10164
    at new Promise (<anonymous>)
    at index-5eb44d2184e903c2.js:1:10132
    at main-7428d4a68ea7713f.js:1:103649
    at Object.next (main-7428d4a68ea7713f.js:1:103754)
    at r (438-4691f28b16d7fe94.js:1:54465)
    at c (438-4691f28b16d7fe94.js:1:54668)
    at 438-4691f28b16d7fe94.js:1:54727
    at new Promise (<anonymous>)
    at 438-4691f28b16d7fe94.js:1:54608
```
No proxies, ad blockers, or extensions in use.

**Comment by drubinstein** [comment]
Hi. Looks like this is related to the recent 0.3.0 release. I was away for a bit, but plan to prioritize fixing this ASAP.

**Comment by drubinstein** [comment]
Fixed, merged and uploaded to pypi as v0.3.2
